be/src/exec/es/es_scroll_parser.cpp

```diff
@@ -275,7 +286,32 @@ int ScrollParser::get_total() {
 Status ScrollParser::fill_tuple(const TupleDescriptor* tuple_desc, 
             Tuple* tuple, MemPool* tuple_pool, bool* line_eof, const std::map<std::string, std::string>& docvalue_context) {
     *line_eof = true;
+
     if (_size <= 0 || _line_index >= _size) {
```

Comment:
maybe _size should be `total` ? currently, the _size = inner_hits.size()

Reply:
i don't think so.
This problem is mainly triggered when _size = 0 and _total > 0。if use `_total`，parser() will return OK() and `_size` remain 0, the problem would not exist.

Reply:
> 
> 
> maybe _size should be `total` ? currently, the _size = inner_hits.size()

i don't think so.
This problem is mainly triggered when _size = 0 and _total > 0。if use `_total`，parser() will return OK() and `_size` remain 0, the problem would not exist.

Reply:
Maybe the concept `size` should keep consistent with before